QUERIES.md

```diff
@@ -37,13 +37,19 @@ REVOKE <privilege> ON <db> FROM <user>
 -- revoke all privileges for a DB
 REVOKE ALL [PRIVILEGES] ON <db> FROM <user>
 
--- revoke all of user's privileges (all DBs and/or cluster admin)
+-- revoke all privileges including cluster admin
 REVOKE ALL [PRIVILEGES] FROM <user>
 
 -- delete a user
 DROP USER <name>
 ```
-where `<privilege> := READ | WRITE | All [PRIVILEGES]`.
+where `<privilege> := READ | WRITE | All [PRIVILEGES]`. 
```

Comment:
@gunnaraasen I removed the optional [PRIVILEGES] from the comment since it adds nothing meaningful. That clarifies the line enough I think.